test/parallel/test-fs-mkdtemp.js

```
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+common.refreshTmpDir();
+
+fs.mkdtempSync(common.tmpDir + '/foo.XXXXXX');
+assert(fs.readdirSync(common.tmpDir).some(function(folder) {
+    return folder.indexOf('foo.') === 0 && folder.length === 'foo.XXXXXX'.length;
+}));
+
+fs.mkdtemp(common.tmpDir + '/bar.XXXXXX', function(err, folder) {
```

Also, don't ignore the error. You can do `assert.ifError(err)`